Formula/yarn.rb

```diff
@@ -18,9 +16,10 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
-    inreplace "#{libexec}/lib/node_modules/yarn/package.json", '"installationMethod": "tar"', '"installationMethod": "homebrew"'
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarn"
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarnpkg"
```

Comment:
I doubt we'll add additional executables other than `yarn` and `yarnpkg` so I think it's fine to just hard-code those two. I don't know enough about Homebrew to have strong feelings either way. 

The executables are in a `bin` directory, could Homebrew just link all the extensionless files in the `bin` directory? https://github.com/yarnpkg/yarn/tree/master/bin